docker/docker_test.go

```diff
@@ -34,8 +33,9 @@ var _ = Describe("Docker", func() {
 
 	Context("Pull", func() {
 		It("can pull a valid container", func() {
-			Expect(pullValidImage()).To(Equal(true))
-			Expect(pullValidImage()).To(Equal(false))
+			// caching in github actions leads to docker image being present from run to run and
+			// failing the previous check
```

Comment:
"failing the previous check" won't make any sense to the previous reader

instead we should have the old checks commented out